3a.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Registration form</title>
+		<link rel="stylesheet" type="text/css" href="final3.css">
+	</head>
+	
+	<body>
+
+
+		<header>
+
+
+			<div class="header">
+             <div class="sub">
```

Comment:
use proper naming convention for the classes name so that it becomes easy to write maintainable css